Cargo.toml

```diff
@@ -15,11 +15,11 @@ homepage = "https://github.com/zkat/big-brain"
 [workspace]
 
 [dependencies]
-bevy = { version = "0.13.0", default-features = false }
+bevy = { version = "0.13.2", default-features = false }
```

Comment:
I'm not sure if it's customary to bump these minor versions in libraries. Local setups usually handle choosing the right version. not sure though

Reply:
Thx for comment. I will happily revert if indeed bad pattern.

Reply:
I tested locally and it looks like it works with either 0.13.0 or 0.13.2. Again, not sure if bad pattern or not, just something I might've seen/heard elsewhere